packages/harmony/src/components/button/SocialButton/SocialButton.module.css

```diff
@@ -0,0 +1,60 @@
+.root {
+  border: transparent;
+  --text-color: var(--harmony-static-white);
+  & svg path {
+    fill: var(--harmony-white);
```

Comment:
do we want the icons to become dark in dark mode? `--harmony-white` will change to a non-white color in other themes

Reply:
Oh interesting. I don't love calling it white then tbh 😅, maybe we should call it `onColor` instead?

Reply:
Yeaaaah def agree there. I understand what they are going for, but I feel like it should have another name. Especially since the other named colors have different values but stay within the category.

We should use a semantic naming here like we do for `--harmony-background`

Light
![image](https://github.com/AudiusProject/audius-protocol/assets/2358254/e19d8f2a-3bc3-4adf-a34a-29f6aca0f8f7)

Dark
![image](https://github.com/AudiusProject/audius-protocol/assets/2358254/13136a52-3e38-41a9-b8dc-74881b93a421)

Reply:
https://www.figma.com/file/ZmEdR1D99gglpBgiZDqWrr/%F0%9F%A7%B1-Foundations?type=design&node-id=1918-4344&mode=dev

looks like this was meant to be called 'harmony-bg-white' which isn't that much better

Reply:
I also think we should probably include the `special-` prefix that the designers have as well 🤔